modules/consensus/consensusset.go

```diff
@@ -50,6 +48,11 @@ type ConsensusSet struct {
 	// up loading blocks from memory.
 	verificationRigor verificationRigor
 
+	// updatePath is a flag that determines if a block node will
+	// be added to the path. It should be false when loading the
+	// current path from disk and true otherwise
+	updatePath bool
```

Comment:
yes, updatePath is there only because blocknodes still need to be run through commitDiffSet, which alters the path. Once all the diffs are saved to disk this won't be necessary.